### ISSUE TITLE
Mobilizing those who want to help?  Collect info starting now?

### ISSUE THREAD

**Issue by JeremyProffitt** [Charlotte-MEDI/Faceshields#12]
I've been getting people asking me how they can help, for now, I'm directing them to go fund me, did we want to make an online web page to collect contact information, area of expertise (use check boxes?), equipment available?  Location?  Then perhaps targeted emails to like people with lasers or 3d printers, etc?

Thoughts on this:

Name:
Email:
Phone:
ZipCode:

Support Effort Areas you can assist in?

3d Printing
Laser Cutting
Assembly
Marketing/Strategy
Medical
Doctor?  Nurse?  Executive?
Materials/Manufacturing

Number of 3d Printers
Number of Laser Cutters

**Comment by DavidCTaylorPhD** [comment]
This is a great idea. This can help organize people. The addresses can be put into google maps to create a list showing where everyone is located visually. Should we add "materials" people have available already?

**Comment by tjfagan** [comment]
I agree, we can work out who is going to be the point person for this in the next meeting.

**Comment by fireroad602** [comment]
Hey! This is Anna Chai. I'm managing the GoFundMe. I am happy to help w. this 👍  

I'm making a master Contacts List of everyone involved, which will help populate a Google Map. 

And -- there is an existing Google Map that lists Industrial 3D Printing Services: 
https://www.google.com/maps/d/u/0/viewer?mid=16agcwfIFD80K9aBFD8DLEpMN3BDPPA2E&ll=35.12360526564431%2C-80.83818775919195&z=9

I would like to reach out to the providers in / around CLT, to see if they want to join CLT_MEDI -- but first, I would like to make a "Getting Started Kit," complete w. a list of materials, sanitizing practices, etc.

**Comment by thomasd538** [comment]
Great idea

**Comment by JeremyProffitt** [comment]
Yep, I'm working today on the 3d printing instructions and video, generic so we can use it for other parts if need be as well

**Comment by fireroad602** [comment]
Thx!! We also need: 

- instructions on disinfecting and sanitizing.

- the modified design files, which Tom is working on.

- a list of materials including quantities ... Shd we also list vendors / sources to purchase?

**Comment by fireroad602** [comment]
I'm adding the new CLT_MEDI social handles to the GoFundMe and will include info about people who want to volunteer and/or get involved and start 3D printing themselves. 

To minimize hand holding, Tom suggested that we vet community makers to people who have a minimum of 5 3D printers. I think this is reasonable. 

I set up cls.makers@gmail.com as a generic e-mail for the GoFundMe but @stoph -- were you able to get up CLT_MEDI@gmail.com ... I'd rather have consistency across all communications.

**Comment by fireroad602** [comment]
Here is the Contact Sheet I started:  https://docs.google.com/spreadsheets/d/165wMwc149RoFIN8_P0QmmomMubuxA5kasyIaaXJE9Fs/edit?usp=sharing

I have been adding notes about offers to help and notes on where we stand w. them. This is editable, so please feel free to add.

**Comment by fireroad602** [comment]
@JeremyProffitt 

Should we be using this? 
3D Printer Crowdsourcing for COVID-19
https://docs.google.com/spreadsheets/u/1/d/1QE1vzezYdf6QJwxaMMLrWIks5AYAPUeSRWXxaYd3yK8/htmlview?sle=true

The Google Sheet is populated by and provides a link to this Submission Form: 
https://docs.google.com/forms/d/e/1FAIpQLSdVfWaQi31l8VNUY6CVctJm5bElMmBKL7YG3mHY8ASvvcwrag/viewform

And they provide Open Source COVID-19 Medical Supplies Guide
https://docs.google.com/document/d/1-71FJTmI1Q1kjSDLP0EegMERjg_0kk_7UfaRE4r66Mg/preview

This might be a good way to add and assess new partners?

**Comment by bramaal** [comment]
Yep! This is good idea everybody must conceive and participate guys
